plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -4420,11 +4450,13 @@ private void fillHostDetailsInfo(VmwareContext serviceContext, Map<String, Strin
         }
     }
 
+    /*
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that this method was not in use. Instead of removing the entire method, I just commented it out in case someone later wanted to easily resume making use of it.